spec/models/user_spec.rb

```diff
@@ -1,5 +1,134 @@
 require 'spec_helper'
 
 describe User do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it { should have_many :members }
+  it { should accept_nested_attributes_for :members }
+  it { should have_many :organizations }
+  it { should have_many :accounts }
+  it { should have_many :movements }
+
+  it { should have_many :posts }
+  it { should have_many :inquiries }
+  it { should have_many :offers }
+
+  it { should validate_presence_of :email }
+  it { should allow_value('my@email.com').for(:email) }
+  it { should_not allow_value('no @ here').for(:email) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.